spec/lib/whitehall_importer/integrity_checker/body_text_check_spec.rb

```diff
@@ -15,25 +15,52 @@
       expect(integrity_check.sufficiently_similar?).to be true
     end
 
-    it "returns true even if there is a mismatch in inline atttachment URL filesize" do
+    it "returns true even if there is a mismatch in an inline attachment URL filesize - with an anchor tag" do
```

Comment:
I don't think we need to specify '- with an anchor tag' here - this type of attachment will always have a link, as per this component: https://components.publishing.service.gov.uk/component-guide/attachment_link

To make a distinction between this and the other style of attachment, we could call this 'attachment links' instead of inline attachments.  It would be consistent with what we call it when inserting an attachment:
<img width="661" alt="Screenshot 2020-03-31 at 11 21 15" src="https://user-images.githubusercontent.com/13434452/78015792-cf86ae00-7341-11ea-9c80-3a09ed3ae95b.png">

Reply:
Thanks @emmabeynon - tests updated.